INSTALLATION.md

```diff
@@ -1,76 +1,78 @@
 
-Two steps are required to use our web on-chain product:
+This document will guide you through the installation process and the first steps to get started with Thyra.
 
-* Thyra installation : obtain the Thyra binary corresponding to your operating system (OS), rename it and make it executable.
-* DNS configuration : install and configure your DNS to resolve the massa top level domain (*.massa) where Thyra runs.
 
-The following installation guide allows you to perform these steps automatically using a simple script to run on your terminal.
-
-Now, let's move on to your OS section:
+> **_TROUBLESHOOTING:_** If you have trouble following this procedure, feel free to [open a question](https://github.com/massalabs/thyra/issues/new) describing your problem.
 
-* [Linux](#linux)
-* [Windows](#windows)
-* [MacOS](#macos)
+## Instructions by Devices
 
+By following these instructions, you will: 
+* Install Thyra to access and perform actions on the blockchain. 
+* Install Thyra application to interact and use Thyra and its plugins very easily.
 
-### MacOS
+### Desktop
 
-Simply use copy/paste the cmd line below in your terminal. The installation process will start and a success message will be displayed once done.
+#### MacOS
 
+1. Go on [this page](https://.com/massalabs/thyra/releases/latest) and download the installer that corresponds to your MacOS configuration
+* If you have M1 chip or more, chose "thyra-installer_darwin_arm64" installer
+* Otherwise, chose "thyra-installer_darwin_amd64" installer
+2. Once downloaded, double clik on the file
+> **_NOTE:_** You may need to authorise the download of this file as your Mac won't recognise it.
+3. A terminal should open automatically where you can follow the steps of the installation. Just watch it working, and at some point you'll need to add your password.
+* Why do you need to enter your password? Simply because Thyra is an application and to add this application to your Mac you need to validate that you wish to install this app; like you do for app from the app store. 
+4. Once the installation is finished, you will see a new icon on the top nav bar (see screenshot below). You can now close your terminal (if not done already) and start browsing the blockchain and all our applications available.
+<img width="177" alt="image" src="https://user-images.githubusercontent.com/109611779/223798813-92a16141-19f6-415b-b9d8-2554eb814edc.png">
 
-```sh
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/massalabs/thyra/main/scripts/macos_install.sh)"
-```
+4. Once the installation is finished, you will see a new icon on the top nav bar. You can now close your terminal (if not done already) and start browsing the blockchain and all our applications available.
```

Comment:
Can be removed I think since there is already a `4.` which is similar to that one before.